test/e2e/view-caseload.js

```diff
@@ -29,6 +29,22 @@ describe('View your caseload flow', () => {
         })
     })
 
+    it('with the correct caseload total summary for each case type', function () {
+      return browser.url(teamCaseloadUrl)
+        .getText('#custodyTotal')
+        .then(function (text) {
+          expect(text).to.eql('315\nCustody cases')
```

Comment:
This number is dependent on seed data, so this test will fail every time we update the seed files? Maybe better to just confirm that a total for each is displayed, and the unit tests will verify it will be the correct total